vnext/Desktop/DevSupportManager.h

```diff
@@ -6,27 +6,21 @@
 #include <ppltasks.h>
 #undef U
 
-#include <IDevSupportManager.h>
-
-#include <boost/asio/ip/tcp.hpp>
-#include <boost/beast/core.hpp>
-#include <boost/system/error_code.hpp>
 #include <cxxreact/JSExecutor.h>
+#include <IDevSupportManager.h>
 
-namespace facebook {
-namespace react {
-
-using JSECreator =
-    std::function<std::unique_ptr<JSExecutor>(std::shared_ptr<ExecutorDelegate>, std::shared_ptr<MessageQueueThread>)>;
+namespace Microsoft::React {
```

Comment:
Is `DevSupportManager` used in devmain? If it is we'd normally want to avoid backporting a namespace change because we want to be able to rev react-native-win32 in older forks, but it sounds like we can't use the latest react-native-win32 DLL for 0.60 in older forks anyway, so this is probably fine.

Reply:
We should ultimately drop all `facebook::react` namespace declarations that actually belong into RNW.

I'm aware of the impact and associated changes it would require on devmain, which we have worked on before.

For this particular change, this isn't an issue, for this class is inside the DLL boundary (not exported).